.gitignore

```diff
@@ -10,4 +10,6 @@ application/cache/*
 
 application/logs/*
 !application/logs/index.html
-!application/logs/.htaccess
\ No newline at end of file
+!application/logs/.htaccess
+/nbproject
```

Comment:
no, to work with netbeans, you need to mention this in the gitignore,
I am checking the test cases, will update the request with the tests working.